vms/platformvm/client.go

```diff
@@ -255,6 +255,8 @@ type Client interface {
 	GetValidatorsAt(ctx context.Context, subnetID ids.ID, height uint64, options ...rpc.Option) (map[ids.NodeID]uint64, error)
 	// GetBlock returns the block with the given id.
 	GetBlock(ctx context.Context, blockID ids.ID, options ...rpc.Option) ([]byte, error)
+	// GetBlockByHeight returns the block at the given [height].
+	GetBlockByHeight(ctx context.Context, height uint64, options ...rpc.Option) ([]byte, error)
```

Comment:
Also, asked in the previous PR. Do we ever need to support multiple heights as input?

Reply:
Unlikely since it's not in the X-chain API set. Outside of the scope of this PR